py/calc_ubi.py

```diff
@@ -235,9 +218,37 @@ def set_ubi(
     )
     if verbose:
         print(f"Adult amount: {gbp(adult_amount / 52)}/week")
+    if pass_income:
+        return basic_income, adult_amount
+    if individual:
+        return (adult_amount/52)
+    else:
+        return adult_amount
+
+
+def set_ubi(
+    base_df, budget, senior, child, dis_1, dis_2, dis_3, regions, verbose=False
+):
+    """Calculate budget-neutral UBI amounts per person.
+
+    Args:
+        base_df (DataFrame): UBI tax reform household-level DataFrame
+        budget (float): Total budget for UBI spending
+        senior (float): Pensioner UBI amount per week
+        child (float): Child UBI amount per week
+        dis_1 (float): Disabled (Equality Act+) supplement per week
+        dis_2 (float): Enhanced disabled supplement per week
+        dis_3 (float): Severely disabled supplement per week
+        regions (ndarray): Numpy float array of 12 UK regional supplements per week
+        verbose (bool, optional): Whether to print the calibrated adult UBI amount. Defaults to False.
+
+    Returns:
+        DataFrame: Reform household-level DataFrame
+    """
+    basic_income, adult_amount = get_adult_amount(base_df, budget, senior, child, dis_1, dis_2, dis_3, regions, pass_income=True)
```

Comment:
Minor suggestions: maybe pass on the verbosity argument?
```suggestion
    basic_income, adult_amount = get_adult_amount(base_df, budget, senior, child, dis_1, dis_2, dis_3, regions, pass_income=True, verbose=verbose)
```